relay-server/src/metrics_extraction/transactions/mod.rs

```diff
@@ -71,57 +71,55 @@ fn extract_geo_country_code(event: &Event) -> Option<String> {
     None
 }
 
-fn is_low_cardinality(source: &TransactionSource, treat_unknown_as_low_cardinality: bool) -> bool {
+fn is_low_cardinality(source: Option<&TransactionSource>) -> bool {
     match source {
         // For now, we hope that custom transaction names set by users are low-cardinality.
-        TransactionSource::Custom => true,
+        Some(TransactionSource::Custom) => true,
```

Comment:
I'm thinking if instead of wrapping all this in `Some` we could use early return? 

```rust

let Some(source) = source else { return false; };

```

and then just match `source` as it was before.